projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/handlers.py

```diff
@@ -144,3 +166,4 @@ def add_handler(handler, endpoint):
     add_handler(DownloadJobHandler, "download")
     add_handler(CreateJobHandler, "create")
     add_handler(LoadJobDataHandler, "job")
+    add_handler(CreateDeploymentHandler, "deploy")
```

Comment:
nit: why CreateDeploymentHandler in favour of DeployJobHandler?
The CLI equivalent of `vdk create` & `vdk deploy`

Reply:
because this command just handles the Deploy create operation. Not remove, update and show ones. You can check https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli.py

Reply:
I see; not sure if it's worth adding a comment hint on this, or too obvious